nemo_aligner/models/nlp/gpt/megatron_gpt_dpo_model.py

```diff
@@ -239,6 +312,14 @@ def get_loss_and_metrics(self, batch, forward_only):
             loss_tensors_list = [loss_reduced["avg"] for loss_reduced in losses_reduced_per_micro_batch]
             loss_tensor = torch.concat(loss_tensors_list)
             loss_mean = loss_tensor.mean()
+            sft_loss_tensors_list = [loss_reduced["avg_sft_loss"] for loss_reduced in losses_reduced_per_micro_batch]
```

Comment:
I didn't update this part since there are some variables like `reward_all_mean, reward_all_std, acc` which cannot be simply put in such a loop without some less prettier if/else statements.